prebuilt/common/bin/bootpwn

```diff
@@ -16,23 +16,21 @@ unset LD_PRELOAD
 
 echo "[!] Welcome to the Pwn zone. Get your game face on."
 
-/system/xbin/busybox stty columns 85
-/system/xbin/busybox stty rows 46
```

Comment:
no replacement for this? I seem to recall this fixed a few things, making the terminfo just a bit cleaner